modules/shared/src/main/scala/io/constellationnetwork/schema/GlobalSnapshotInfo.scala

```diff
@@ -219,7 +230,8 @@ case class GlobalSnapshotInfo(
   activeDelegatedStakes: Option[SortedMap[Address, List[DelegatedStakeRecord]]],
   delegatedStakesWithdrawals: Option[SortedMap[Address, List[PendingDelegatedStakeWithdrawal]]],
   activeNodeCollaterals: Option[SortedMap[Address, List[NodeCollateralRecord]]],
-  nodeCollateralWithdrawals: Option[SortedMap[Address, List[PendingNodeCollateralWithdrawal]]]
+  nodeCollateralWithdrawals: Option[SortedMap[Address, List[PendingNodeCollateralWithdrawal]]],
+  priceState: Option[SortedMap[CurrencyId, PriceRecord]]
```

Comment:
not super familiar with the reqs for this but are we wanting to track the price of any ML0 or just for DAG?